python/pyspark/sql/connect/expressions.py

```diff
@@ -490,7 +490,7 @@ def __repr__(self) -> str:
                 # is sightly different:
                 # java.time.Duration only applies HOURS, MINUTES, SECONDS units,
                 # while Pandas applies all supported units.
-                return pd.Timedelta(delta).isoformat()  # type: ignore[attr-defined]
```

Comment:
it is surprising that the two `type: ignore` were caused by non-linter dependency.

Reply:
Oh, good to know. I also didn't expect those kind of situations.